scipy/stats/stats.py

```diff
@@ -5907,8 +5908,12 @@ def _permutation_ttest(a, b, permutations, axis=0, equal_var=True,
     b = mat_perm[..., na:]
     t_stat = _calc_t_stat(a, b, equal_var)
 
+    compare = {"less": np.less_equal,
+               "greater": np.greater_equal,
+               "two-sided": lambda x, y: np.abs(x) >= np.abs(y)}
```

Comment:
maybe I haven't done a hypothesis test is a few years, why do we need `abs()` here?

Reply:
To do the two sided tests, we're looking for how much of the distribution under the null hypothesis is more extreme than the observed test statistic, so we want to do:
`x <= y | x >= y`
I did:
`np.abs(x) >= np.abs(y)}`
instead.
Whichever you prefer.

Reply:
> To do the two sided tests, we're looking for how much of the distribution of the statistic under the null hypothesis is more extreme than the observed test statistic. Since that distribution is symmetric about the origin, we want to count:
> `x <= -np.abs(y) | x >= np.abs(y)`
> Equivalently, I did:
> `np.abs(x) >= np.abs(y)`
> instead.
> Now that you mention it, I wouldn't mind `x <= -np.abs(y) | x >= np.abs(y)` if you'd like me to change it.

yeah, I think it might be more obvious on a quick read to write it this longer way even though they're equivalent.

Reply:
```suggestion
               "two-sided": lambda x, y: (x <= -np.abs(y)) | (x >= np.abs(y))}
```